ingest/expression_files/mtx.py

```diff
@@ -116,6 +151,23 @@ def check_duplicates(names: List, file_type: str):
             raise ValueError(msg)
         return True
 
+    @staticmethod
+    def get_data_start_line_number(file_path: str) -> int:
+        """ Determines what line number data starts.
+
+         Returns
+         ----------
+            i (int) - Line number where data starts
+        """
+        i = 1
+        for line in file_path:
+            if line.startswith("%"):
+                i = +1
+            else:
+                # First line w/o '%' is mtx dimension. So skip this line (+2)
+                return i + 2
```

Comment:
Why are we skipping two lines once we reach the matrix dimensions?  Shouldn't we only skip one more line?

Reply:
No. Because we haven't incremented by one for the first line that didn't have the %. Then you add 1 more to get past the MTX dimensions.

Reply:
You're right! 3007fc0